### DISCUSSION TITLE
建议添加零冷水热水器一键即热控制开关

### DISCUSSION THREAD

**Discussion by goven** [XiaoMi/ha_xiaomi_home#134] [Ideas]
建议添加零冷水热水器一键即热控制开关，目前只能调整水温。

**Comment by SusanPhevos** [comment]
您好！请问能否发一下Home Assistant设备详情页面？我看看是不是一个问题。

**Reply by goven** [reply]
![image](https://github.com/user-attachments/assets/b1ab70bf-4927-4004-8972-5deb9822ad3c)

![image](https://github.com/user-attachments/assets/00076174-8607-481d-bfae-ab641343ce5a)

**Comment by goven** [comment]
![image](https://github.com/user-attachments/assets/54298b63-7e1b-44d4-9f9a-9ff980b46ddb)

**Comment by SusanPhevos** [comment]
xiaomi_home/miot/specs/spec_add.json 的第 2 行添加以下内容（注意末尾的逗号）：
```
  "urn:miot-spec-v2:device:water-heater:0000A02A:viomi-u10:1": [
    {
      "iid": 2,
      "type": "urn:miot-spec-v2:service:heater-machine:0000783E:viomi-u10:1",
      "description": "Water Heater",
      "properties": [
        {
          "iid": 4,
          "type": "urn:miot-spec-v2:property:on:00000006:viomi-u10:1",
          "description": "Switch Status",
          "format": "bool",
          "access": [
            "read",
            "write"
          ]
        }
      ]
    }
  ],
```
修改文件后，重启 Home Assistant，选择 xiaomi_home > 配置 > 更新实体转换规则，可在热水器实体之外多生成一个开关实体，这个实体对应 [viomi.waterheater.u10](http://poc.miot-spec.srv/miot-spec-v2/instance?type=urn:miot-spec-v2:device:water-heater:0000A02A:viomi-u10:1) 的 siid=2 piid=4 开关。
<img width="652" height="249" alt="image" src="https://github.com/user-attachments/assets/96d0ebe3-b5c6-4b91-9964-9c786f008be4" />
